docs/sources/panels-visualizations/query-transform-data/transform-data/index.md

```diff
@@ -761,13 +761,13 @@ Use this transformation to merge multiple results into a single table, enabling
 
 This is especially useful for converting multiple time series results into a single wide table with a shared time field.
 
-#### Inner join
+#### Inner join (for Time Series or SQL-like data)
```

Comment:
i'm wondering why we have two outer join options and only one inner join 🤔 maybe the outer join should also be consolidated to reduce option overload / need for user to care about the shape of their data?
<img width="1130" alt="Screenshot 2024-05-08 at 4 35 25 PM" src="https://github.com/grafana/grafana/assets/22381771/2720cf91-923a-4144-abe6-4ee0abd1bed3">

Reply:
the timeseries outer join algorithm sorts the values, and does not handle duplicate timestamps in the same series. this makes it really fast but not a true sql outer join. i dont think we can make a sql outer join work as fast, but would love to be wrong here.